packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -28,12 +28,35 @@ const reducerMap = {
             cartId: String(payload)
         };
     },
+    [actions.mergeCarts.request]: state => {
+        return {
+            ...state,
+            isLoading: true
+        };
+    },
     [actions.getDetails.request]: state => {
         return {
             ...state,
             isLoading: true
         };
     },
+    [actions.mergeCarts.receive]: (state, { payload, error }) => {
```

Comment:
Move the `request`/`receive` functions next to their counterparts. Right now you have request, request, receive, receive.

Reply:
Removed usage of actions.